keyboards/ploopyco/trackball_nano/keymaps/viam/keymap.c

```diff
@@ -25,8 +25,16 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 void suspend_power_down_user(void) {
     // Switch off sensor + LED making trackball unable to wake host
     adns5050_power_down();
+
+    #if (defined(BETTER_DRAGSCROLL) && defined(BETTER_DRAGSCROLL_ON_HOST_VALUE_TOGGLED) && (defined(BETTER_DRAGSCROLL_SCRLK_ENABLE) || defined(BETTER_DRAGSCROLL_CAPLK_ENABLE) || defined(BETTER_DRAGSCROLL_NUMLK_ENABLE)) || (defined(VIA_ENABLE) && defined(PLOOPY_VIAMENUS)))
```

Comment:
Given that this fork is meant to be used with vial only, does it really matter to have all these ifdefs?

Unless you want to get this merged upstream... But I assume that also via builds are assuming that `VIA_ENABLED` is always set, no?